goalc/compiler/compilation/Asm.cpp

```diff
@@ -370,9 +562,10 @@ Val* Compiler::compile_asm_blend_vf(const goos::Object& form, const goos::Object
 Val* Compiler::compile_asm_vf_math3(const goos::Object& form,
                                     const goos::Object& rest,
                                     IR_VFMath3Asm::Kind kind,
+                                    emitter::Register::XMM_ELEMENT broadcastElement,
                                     Env* env) {
   auto args = get_va(form, rest);
-  va_check(form, args, {{}, {}, {}}, {{"color", {false, goos::ObjectType::SYMBOL}}});
+  va_check(form, args, {{}, {}, {}, {}}, {{"color", {false, goos::ObjectType::SYMBOL}}});
```

Comment:
Same comment as the `abs`, we could make this a `:mask` keyword argument. Feel free to leave this for later.